cpp/kiss_icp/core/Deskew.hpp

```diff
@@ -32,6 +32,7 @@ namespace kiss_icp {
 std::vector<Eigen::Vector3d> DeSkewScan(const std::vector<Eigen::Vector3d> &frame,
                                         const std::vector<double> &timestamps,
                                         const Sophus::SE3d &start_pose,
-                                        const Sophus::SE3d &finish_pose, int prev_frame_delta=1);
+                                        const Sophus::SE3d &finish_pose,
+                                        float frame_drop_ratio = 1);
```

Comment:
Why did you choose **float** as the data type? I mean KISS seems like to use **double** everywhere else, so I would go with **double**.

Reply:
They use float. For example 
python/kiss_icp/config/config.py:    max_range: float = 100.0

I see they use double for timestamp which makes sense. It needs accuracy. For ratio float is enough.

Reply:
I don't have a strong opinion since the net effect is about the same, I just recommend **double** because I believe that the primitive type of `Sophus::SE3d` to be **double**